From: Markus Koschany <apo@debian.org>
Date: Thu, 15 Apr 2021 00:26:17 +0200
Subject: ignore TestSSLSocketFactory.testTLSOnly

Ignore the failing testTLSOnly test because it deliberately sets the protocol
version to TLSv1 which is disabled by default via the java.security policy in
OpenJDK 11. TLSv1 is considered to be no longer safe and rejecting the
handshake is actually the correct behavior.

Forwarded: no
---
 .../src/test/java/org/apache/http/conn/ssl/TestSSLSocketFactory.java    | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/httpclient/src/test/java/org/apache/http/conn/ssl/TestSSLSocketFactory.java b/httpclient/src/test/java/org/apache/http/conn/ssl/TestSSLSocketFactory.java
index 3865f16..725a5d9 100644
--- a/httpclient/src/test/java/org/apache/http/conn/ssl/TestSSLSocketFactory.java
+++ b/httpclient/src/test/java/org/apache/http/conn/ssl/TestSSLSocketFactory.java
@@ -58,6 +58,7 @@ import org.apache.http.ssl.SSLContexts;
 import org.hamcrest.CoreMatchers;
 import org.junit.After;
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
@@ -290,6 +291,7 @@ public class TestSSLSocketFactory {
         sslSocket.close();
     }
 
+    @Ignore
     @Test
     public void testTLSOnly() throws Exception {
         // @formatter:off
